lichen_cli/src/main.rs

```diff
@@ -200,9 +201,40 @@ fn main() -> color_eyre::Result<()> {
     set_colors_enabled(true);
 
     let euid = unsafe { geteuid() };
-    if euid != 0 {
-        bail!("lichen must be run as root. Re-run with sudo")
-    }
+    ensure!(euid == 0, "lichen must be run as root. Re-run with sudo.");
+
+    let partition_detection_warning = indoc! {"
+        The installer currently does not attempt to detect if there is a file system
+        on detected ESP (and XBOOTLDR) partitions.
+        
```

Comment:
Empty lines with spaces are pretty unusual - I guess rustfmt doesn't complain because this is inside a string literal so it actually makes a difference (except the macro erases the difference again).